config/routes.rb

```diff
@@ -28,6 +28,7 @@
     end
     member do
       post :review
+      post :rejected
```

Comment:
It rubs me the wrong way that this action is in the past tense. When you call it you're actively rejecting a submission, not looking for ones that have been previously rejected.

Reply:
ok ill change it